Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -2747,6 +2755,10 @@ void WebPage::setDeviceScaleFactor(float scaleFactor)
     updateHeaderAndFooterLayersForDeviceScaleChange(scaleFactor);
 #endif
 
+#if USE(SKIA)
+    FontRenderOptions::singleton().setUseSubpixelPositioning(scaleFactor > 1.);
```

Comment:
What if the scale factor is close to 1.0, let's say 1.1? Wouldn't it be better to increase this threshold a bit?

Reply:
Good point, I'm again doing what chromium does, but I guess they don't support fractional scaling, so `> 1` means `<= 2`.